tests/test_qnode.py

```diff
@@ -680,6 +680,19 @@ def func(x, y):
         ):
             qn(5, 1)
 
+        def func(x, y):
+            qml.RX(x, wires=0)
+            qml.RY(y, wires=1)
+            qml.CNOT(wires=[0, 1])
+            return []
+
+        qn = QNode(func, dev)
+
+        with pytest.raises(
+            qml.QuantumFunctionError, match="must return either a single measurement"
+        ):
+            qn(5, 1)
+
```

Comment:
Since the test has been updated, I think it should be linted and added to `tests_passing_pylint.txt`

Reply:
there are ~80 violations, I don't think it's worth my time tbh 😄

Reply:
At that many, agree.